node-playwright/main.js

```diff
@@ -25,14 +25,14 @@ Apify.main(async () => {
 
     })
 
-    const promisesHeadful = browsers.map( async browserName => {
+    const promisesHeadfull = browsers.map( async browserName => {
```

Comment:
Actually, `headful` is correct :) It's same as stateless vs. stateful

Reply:
Ok, thanks!